softlayer/resource_softlayer_scale_group.go

```diff
@@ -131,6 +131,31 @@ func resourceSoftLayerScaleGroup() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeInt},
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					o := strings.TrimRight(strings.TrimLeft(old, " ["), "] ")
```

Comment:
As discussed in the scrum, we can try to address this by simply checking `n` and checking whether it is already a member of `d.Get('network_vlan_ids')`. If true, we suppress, otherwise we don't suppress.